experimental/packages/otlp-transformer/package.json

```diff
@@ -77,10 +115,11 @@
     "nyc": "15.1.0",
     "protobufjs-cli": "1.1.3",
     "ts-loader": "9.5.1",
-    "typescript": "4.4.4",
+    "typescript": "5.7.2",
```

Comment:
I had to upgrade to Typescript v5 for `typebox` to work properly. Since this PR is targeting the `next` branch which is meant to allow newer typescript versions, I assume this isn't an issue (although I'm not sure which specific typescript v5 version you want to use)